### PR TITLE
Support dbt aliases

### PR DESCRIPTION
**Pull request by fivetran-dmitryboykov**

### CODE REVIEW

File: server/src/ZetaSqlWrapper.ts [hunk 1]
```diff
@@ -65,7 +65,12 @@ export class ZetaSqlWrapper {
   }
 
   getTableRef(model: ManifestModel, name: string): string[] | undefined {
-    return model.refs.find(ref => ref.indexOf(name) === ref.length - 1);
+    const tableRef = model.refs.find(ref => ref.indexOf(name) === ref.length - 1);
+    if (tableRef) {
+      return tableRef;
+    }
+    const aliasedModel = this.dbtRepository.models.find(m => m.alias === name);
+    return aliasedModel && model.refs.some(ref => ref.indexOf(aliasedModel.name) === ref.length - 1) ? [aliasedModel.name] : undefined;
```
Comment (pgrivachev) [inline server/src/ZetaSqlWrapper.ts:73]: There is duplication here:
`model.refs.some(ref => ref.indexOf(aliasedModel.name) === ref.length - 1)`

File: e2e/src/dbt_alias.spec.ts [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+suite('Should resolve tables with specified alias', () => {
+  const USERS_FILE_NAME = `${COMPLETION_JINJA_PATH}/models/users.sql`;
+  const JOIN_REF_FILE_NAME = `${COMPLETION_JINJA_PATH}/models/join_ref.sql`;
+  const USERS_DOC_URI = getCustomDocUri(USERS_FILE_NAME);
+  const JOIN_REF_DOC_URI = getCustomDocUri(JOIN_REF_FILE_NAME);
+
+  test('Should resolve table introduced by model with specified alias', async () => {
+    await activateAndWait(USERS_DOC_URI);
+    await replaceText("materialized='table'\n  )\n}}", "materialized='table', alias='super_users'\n  )\n}}\n--Comment");
```
Comment (pgrivachev) [inline e2e/src/dbt_alias.spec.ts]: Why do we need `--Comment` at the end?

File: e2e/src/helper.ts [hunk 3]
```diff
@@ -54,6 +54,11 @@ const languageServerReady = new Array<[string, DeferredResult<void>]>();
 
 let tempModelIndex = 0;
 
+export async function openDocument(docUri: Uri): Promise<void> {
```
Comment (pgrivachev) [inline e2e/src/helper.ts:57]: Can you replace the following calls to this function call everywhere?
```
  doc = await workspace.openTextDocument(newUri);
  editor = await window.showTextDocument(doc);
```

**Review by pgrivachev — APPROVED** [review]
Looks good! It just needs a little refactoring.

File: e2e/src/dbt_alias.spec.ts (hunk 2 above)
Comment (fivetran-dmitryboykov) [reply e2e/src/dbt_alias.spec.ts]: It was needed previously to update Preview.

**Review by pgrivachev — APPROVED** [review]

**Pull request merged**